public/app/core/components/SafeDynamicImport.tsx

```diff
@@ -1,20 +1,48 @@
 import React, { lazy, Suspense, FunctionComponent } from 'react';
-import { cx } from 'emotion';
-import { ErrorBoundaryAlert, LoadingPlaceholder } from '@grafana/ui';
+import { cx, css } from 'emotion';
+import { LoadingPlaceholder, ErrorBoundary, Button } from '@grafana/ui';
 
 export const LoadingChunkPlaceHolder: FunctionComponent = () => (
   <div className={cx('preloader')}>
-    <LoadingPlaceholder text={'Loading chunk...'} />
+    <LoadingPlaceholder text={'Loading...'} />
   </div>
 );
 
+function getAlertPageStyle() {
```

Comment:
Is there an advantage in having this as a function, versus just a variable?

Reply:
Unfortunately just cut & paste in this case because I was in a hurry.